src/main/thrift/parquet.thrift

```diff
@@ -738,10 +738,10 @@ struct PageHeader {
 }
 
 /**
- * Wrapper struct to specify sort order
+ * Sort order within a RowGroup
  */
 struct SortingColumn {
-  /** The column index (in this row group) **/
+  /** The ordinal position of the column (in this row group) **/
```

Comment:
I'd suggest to explicitly say it is a leaf column.

Reply:
in c7e3ae7